test/protractor.conf.js

```diff
@@ -1,11 +1,5 @@
 exports.config = {
   seleniumAddress: 'http://localhost:4444/wd/hub',
   specs: ['e2e/*.js'],
-  baseUrl: 'http://localhost:8080',
-  onPrepare: function(){
-    require('protractor-http-mock').config= {
-      rootDirectory: process.cwd(),
-      protractorConfig: 'test/protractor.conf.js'
-    };
-  }
+  baseUrl: 'http://localhost:8080'
 };
```

Comment:
If you want to make it pretty - not just have dots
```npm install jasmine-spec-reporter --save-dev```

then follow the steps here: https://github.com/bcaudan/jasmine-spec-reporter/blob/master/docs/protractor-configuration.md